src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -1158,7 +1162,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
 
         if (cmdChannel.getConverter() == null) {
-            logger.warn("NODE {}: No command converter set for command {} type {}", nodeId, channelUID, dataType);
+            logger.debug("NODE {}: No command converter set for command {} type {}", nodeId, channelUID, dataType);
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.